dev-util/nvidia-cuda-sdk/nvidia-cuda-sdk-8.0.44.ebuild

```diff
@@ -0,0 +1,146 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cuda eutils flag-o-matic portability toolchain-funcs unpacker versionator
+
+MYD=$(get_version_component_range 1-2)
+
+DESCRIPTION="NVIDIA CUDA Software Development Kit"
+HOMEPAGE="https://developer.nvidia.com/cuda-zone"
+SRC_URI="https://developer.nvidia.com/compute/cuda/${MYD}/prod/local_installers/cuda_${PV}_linux-run -> cuda_${PV}_linux.run"
+
+LICENSE="CUDPP"
+SLOT="0"
+KEYWORDS="~amd64 ~amd64-linux"
+IUSE="debug +doc +examples opencl +cuda"
+
+RDEPEND="
+	~dev-util/nvidia-cuda-toolkit-${PV}
+	media-libs/freeglut
+	examples? (
+		media-libs/freeimage
+		media-libs/glew:*
+		virtual/mpi
+		>=x11-drivers/nvidia-drivers-367.44[uvm]
+		)"
+DEPEND="${RDEPEND}"
+
+RESTRICT="test"
+
+S=${WORKDIR}/samples
+
+QA_EXECSTACK=(
+	opt/cuda/sdk/0_Simple/cdpSimplePrint/cdpSimplePrint
+	opt/cuda/sdk/0_Simple/cdpSimpleQuicksort/cdpSimpleQuicksort
+	opt/cuda/sdk/bin/x86_64/linux/release/cdpSimplePrint
+	opt/cuda/sdk/bin/x86_64/linux/release/cdpSimpleQuicksort
+	)
+
+src_unpack() {
+	unpacker
+	unpacker run_files/cuda-samples*run
```

Comment:
Also, you should check the underlying compression. If it happens to be xz, then you need to dep on xz-utils.